### PR TITLE
Issue#7392 , Updated Prerequisites for meshery docs.

### LINKED ISSUE #7392: [Docs] Outdated Prerequisites mentioned in Meshery Docs
#### Current State
Steps to Reproduce: 
1. Go to: https://docs.meshery.io/project/contributing
2. Scroll Down to `Prerequisites for building Meshery in your development environment`

Here you will see that the prerequisites mentioned are wrong and should be updated 
- Wrong Meshery Repository link mentioned.
- Right now Node version 19 is not supported 
- Golang version 1.20 is not supported
Screenshot from 2023-03-26 02-33-46

#### Desired State

The instructions should be updated with the supported versions of the required prerequisites.

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by hirentimbadiya**
**Notes for Reviewers**

This PR fixes #7392 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e8b57fb4c07bd5030b1d0d342f63bed10a8845d5 at: https://642aa2a4a6d5895a7c012312--meshery-docs-preview.netlify.app

**Comment by hirentimbadiya** [comment]
@leecalcote , @thisiskaransgit , @Chadha93 , @Nikhil-Ladha  please review this PR.

File: docs/pages/project/contributing/contributing.md [hunk 1]
```diff
@@ -86,8 +86,7 @@ Practices for Production Environments](https://peter.bourgon.org/go-in-productio
...
 
-1. `Go` version 1.11+ installed if you want to build and/or make changes to the existing code.
+1. `Go` version 1.19 must be installed if you want to build and/or make changes to the existing code.
 1. `GOPATH` environment variable should be configured appropriately
-1. `npm` and `node` should be installed on your machine, preferably the latest versions.
-1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have a version of `Go` prior to version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
-
+1. `npm` and `node` should be installed on your machine, `node` version 19 or higher are not supported right now.
+1. Fork this repository (`git clone https://github.com/meshery/meshery`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have a version of `Go` prior to version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
```
Comment (hexxdump) [inline docs/pages/project/contributing/contributing.md:92]: Line number 1 states, go must be of version 1.19. So the second half of the line about go version 1.13 does not make sense to me, Am I missing something ?

// @thisiskaransgit

**Comment by thisiskaransgit** [comment]
@hirentimbadiya 
Update it with mentioned instructions.
 
 1. Go version 1.19 must be installed if you want to build and/or make changes to the existing code. The binary `go1.19` should be available in your path. If you don't want to disturb your existing version of Go, then follow these [instructions](https://go.dev/doc/manage-install#:~:text=and%20run%20them.-,Installing%20multiple%20Go%20versions,-You%20can%20install) to keep multiple versions of Go in your system.
 1. `GOPATH` environment variable should be configured appropriately
 1. `npm` and `node` should be installed on your machine, node version 19 or higher is not supported right now.
 1. Fork this repository (`git clone https://github.com/meshery/meshery.git`), and clone your forked version of Meshery to your development environment, preferably outside `GOPATH`.
1. `golangci-lint` should be installed if you want to test Go code, for MacOS and linux users.

File: docs/pages/project/contributing/contributing.md (hunk 1 above)
Comment (thisiskaransgit) [reply docs/pages/project/contributing/contributing.md:92]: Keen eyes @hexxdump, thank you, @hirentimbadiya you should remove that line.

Comment (hirentimbadiya) [reply docs/pages/project/contributing/contributing.md:92]: okay @thisiskaransgit and thanks for reviewing @hexxdump.

**Review by thisiskaransgit — APPROVED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit c1bf1d8d1b5106e6b0597b8b8119377ef69d9218 at: https://642af9c9f88833389063cf26--meshery-docs-preview.netlify.app

**Review by hexxdump — APPROVED** [review]
LGTM

**Pull request merged**